README.md

```diff
@@ -39,20 +39,72 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ```
 
-## Gems
+## Your local development environment
+Before you get started you should ensure your development machine has the following files located under `~/.aaf`.
+
+1. `rapidconnect.yml`
+
+	Generate a secret to use with Rapid Connect with the following: 
+		
+		$> tr -dc '[[:alnum:][:punct:]]' < /dev/urandom | head -c32 ;echo
+		
+	Access [https://rapid.test.aaf.edu.au](https://rapid.test.aaf.edu.au) and register using your secret created above and the callback URL of `http://localhost:8080/auth/jwt`
```

Comment:
It needs to be an `auresearch` service for most/all of our stuff to work. Currently this can only be done by a rapidconnect admin.